opentreemap/treemap/templates/treemap/partials/add_plot_btn.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
```

Comment:
I think the name right now is accurate enough - adding a tree necessarily entails adding a plot as well, and in fact the add process won't create a tree if you do not fill in tree fields.

Also, I don't think changing the name is enough of an improvement to refactor the code that is relying on the current value of the `data-feature` attribute.